src/pages/index.js

```diff
@@ -82,22 +93,53 @@ function getCardElement(data) {
   const cardElement = cardTemplate.content
     .querySelector(".card")
     .cloneNode(true);
+  cardElement._id = data._id;
 
   const cardTitle = cardElement.querySelector(".card__title");
   const cardImage = cardElement.querySelector(".card__image");
   const cardLikeButton = cardElement.querySelector(".card__like-button");
+  const cardLikeCount = cardElement.querySelector(".card__like-count");
   const cardDeleteButton = cardElement.querySelector(".card__delete-button");
 
   cardTitle.textContent = data.name;
   cardImage.alt = data.name;
   cardImage.src = data.link;
 
+  // Set initial like state and count
+  cardLikeCount.textContent = data.likes ? data.likes.length : 0;
+  if (data.likes && data.likes.some((like) => like._id === userId)) {
```

Comment:
`  if (data.likes && data.likes.some((like) => like._id === userId)) {
`

* When I reload the page, all **red** hearts become **white** again. Please, check `isLiked` when you create a card

```js

  if (data.isLiked) {
    cardLikeButton.classList.add("card__like-button_liked");
  }

```